ruma-events/src/lib.rs

```diff
@@ -520,6 +520,18 @@ pub struct EventDeHelper {
     pub unsigned: Option<UnsignedDeHelper>,
 }
 
+/// Helper struct to determine the msgtype from a `serde_json::value::RawValue`
+#[doc(hidden)]
+#[derive(Debug, Deserialize)]
+pub struct MessageDeHelper {
```

Comment:
This type should be in `room::message` (or `room::message::content_serde`) and not be public. The fields also don't need `pub` then.